### ISSUE TITLE
[Feature Request] [3.0.0-beta.6] VAlert should have an accessible close action

### ISSUE THREAD

**Issue by paul-thebaud** [vuetifyjs/vuetify#15589]
### Problem to solve
Currently, the VAlert component close action is not accessible: it is not keyboard focusable and does not have any textual alternative.

### Proposed solution
I think there are two available solutions to solve this:

- Add a `role="button"`, `tabindex="1"` and `aria-label="Close"` to the `div.v-alert__close`
- Use a `v-btn` with appropriate settings to match alert style (probably better IMO)

I can create a PR if needed, but I would like to know which solution is preferred by the core team.

**Comment by paul-thebaud** [comment]
Hi @johnleider and @KaelWD, I think this is probably applicable to other components (in Vuetify, this was the case on clearable inputs, calendar and time pickers, etc.).
Should I open a new issue?
Btw, thanks for the great work again!

**Comment by KaelWD** [comment]
> clearable inputs

#15641

> calendar and time pickers

Don't currently exist, should hopefully be done correctly when they do.

**Comment by paul-thebaud** [comment]
Great, thanks for taking the time to link those!
